Modules/Files/test/Files.API.Tests.Performance.EndToEnd/src/get-metadata.test.ts

```diff
@@ -0,0 +1,77 @@
+import http from "k6/http";
+import { Options } from "k6/options";
+import {
+  describe,
+  expect,
+} from "https://jslib.k6.io/k6chaijs/4.3.4.2/index.js";
+import { getJwt, assertEnvVarExists, Size, tomorrow } from "./utils";
+
+assertEnvVarExists();
+
+const apiEndpoint = __ENV.HOST + "/api/v1";
+
+interface Data {
+  authToken: string;
+  fileId: string;
+}
+
+function size(): Size {
+  switch (__ENV.SIZE) {
+    case "S":
+      return { vus: 1, iterations: 10 };
+    case "M":
+      return { vus: 10, iterations: 50 };
+    case "L":
+      return { vus: 50, iterations: 100 };
+    default:
+      throw new Error("Invalid 'Size' value: " + __ENV.SIZE);
+  }
+}
+
+export const options: Options = {
+  vus: size().vus,
+  thresholds: {
+    http_req_duration: ["p(90)<50", "p(98)<100"],
+  },
+  iterations: size().iterations,
+};
+
+export function setup(): Data {
+  const authToken = getJwt();
```

Comment:
Maybe we should call the method `getAuthenticationHeader` and return `Bearer ${JWT}`, so that we don't have to concatenate it everywhere.